From 8a8f3a307b77d0aeed65524467a89ce1e30911b6 Mon Sep 17 00:00:00 2001
From: Manoj Srivastava <srivasta@golden-gryphon.com>
Date: Sun, 28 Apr 2013 02:48:57 -0700
Subject: [PATCH] [topic--man-fix]: spelling errors in man pages

seperately separately
explicitely explicitly
noticable noticeable
enviroment environment

Signed-off-by: Manoj Srivastava <srivasta@golden-gryphon.com>
(cherry picked from commit ca33ddc478e32e645ad8fce4760a6d8fbfd8cda1)
---
 Changes                            | 4 ++--
 lib/Log/Log4perl/Appender/Limit.pm | 2 +-
 t/026FileApp.t                     | 2 +-
 t/028Additivity.t                  | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/Changes b/Changes
index 1ee33b3..4b1b9ba 100644
--- a/Changes
+++ b/Changes
@@ -786,7 +786,7 @@
            still be called __ANON__, but this can be overridden by 
            defining
                local *__ANON__ = "subroutine_name";
-           in them explicitely (thanks, Perlmonks :).
+           in them explicitly (thanks, Perlmonks :).
 
 0.41 (12/12/2003)
     * (ms) Applied documentation update for Synchronized appender, suggested
@@ -845,7 +845,7 @@
     * (ms) Matthew Keene <mkeene@netspace.net.au> suggested to have
       an accessor for all appenders currently defined -- added 
       appenders() method
-    * (ms) Test case 041SafeEval.t didn't share $0 explicitely and
+    * (ms) Test case 041SafeEval.t didn't share $0 explicitly and
       created some warnings, fixed that with (jf)'s help.
     * (ms) Added performance improvements suggested by
       Kyle R. Burton <mortis@voicenet.com>. is_debug/is_info/etc.
diff --git a/lib/Log/Log4perl/Appender/Limit.pm b/lib/Log/Log4perl/Appender/Limit.pm
index 8c55907..2f5c8a6 100644
--- a/lib/Log/Log4perl/Appender/Limit.pm
+++ b/lib/Log/Log4perl/Appender/Limit.pm
@@ -221,7 +221,7 @@ __END__
     WARN("This message will be sent immediately.");
     WARN("This message will be delayed by one hour.");
     sleep(3601);
-    WARN("This message plus the last one will be sent now, seperately.");
+    WARN("This message plus the last one will be sent now, separately.");
 
 =head1 DESCRIPTION
 
diff --git a/t/026FileApp.t b/t/026FileApp.t
index 560f260..f333270 100644
--- a/t/026FileApp.t
+++ b/t/026FileApp.t
@@ -72,7 +72,7 @@ close FILE;
 
 ####################################################
 # Append to a log file without specifying append mode
-# explicitely
+# explicitly
 ####################################################
 my $data = <<EOT;
 log4j.category = INFO, FileAppndr
diff --git a/t/028Additivity.t b/t/028Additivity.t
index 118d09a..7dbbff1 100644
--- a/t/028Additivity.t
+++ b/t/028Additivity.t
@@ -72,7 +72,7 @@ ok($buf2, "INFO - Percolate this!\n");
 Log::Log4perl::Appender::TestBuffer->reset();
 
 ######################################################################
-# Define the root logger and another logger, additivity on explicitely
+# Define the root logger and another logger, additivity on explicitly
 ######################################################################
 Log::Log4perl->init(\<<'EOT');
     log4perl.logger = INFO, A1
